src/components/Tool/Appeal/AppealsContext/AppealsContext.tsx

```diff
@@ -269,22 +269,30 @@ export const AppealsProvider: React.FC<AppealsContextProps> = ({
     debounce((searchTerm: string) => {
       const { searchTerm: _, ...oldQuery } = query;
       if (searchTerm !== '') {
-        replace({
-          pathname,
-          query: {
-            ...oldQuery,
-            accountListId,
-            ...(searchTerm && { searchTerm }),
+        replace(
```

Comment:
Great find! There might be some other places that could benefit from this in the future.

Reply:
Thanks